### PR TITLE
Add new Block syntax

### PR DESCRIPTION
**Pull request by classerase**
This pull request will add the new block syntax:
var xx = 1
var vv = "Hello, world!"
var foo = { vv, xx }

It also adds a symlink command (currently only usable in linux)

This Pull-Request also changes `Token.DblDot` to `Token.Colon`

### CODE REVIEW

**Comment by PeyTy** [comment]
Seems a bit broken! You may create a separate PR with Colon and symlink features 😉

**Comment by PeyTy** [comment]
Or just revert changes to Parser in this PR

**Comment by classerase** [comment]
i reverted the changes in parser.hexa. i will investigate more locally and then open a new pull request

File: source/main.hexa [hunk 1]
```diff
@@ -178,6 +178,18 @@ class Main {
 
 		Process.stdout.write("Hexa package initialized!")
 	}
+	
+	function symlink(): Void {
+		switch (process.platform) {
+			case 'linux':
+				// TODO probably just copy or do actual symlinking?
```
Comment (PeyTy) [inline source/main.hexa:185]: Edited a bit

Comment (PeyTy) [reply source/main.hexa:185]: @classerase any ideas?

Comment (classerase) [reply source/main.hexa:185]: hmm..

**Pull request merged**